src/pages/myGeotab/whatsNew.tsx

```diff
@@ -1,8 +1,3045 @@
+		<ul>
+			<li>Added Add-In samples for the AddInData API to the <a href="https://github.com/Geotab/sdk-addin-samples">Geotab/sdk-addin-samples</a> Github repository.</li>
+		</ul>
+	</div>
+);
+
+//ToDo Update Concepts Page Link
+const update9: ReactNode = (
+	<div className="paragraph">
+		<h2>Custom Security Identifiers For Add-Ins</h2>
+		<p>Custom security identifiers for MyGeotab Add-ins are now available. Identifiers can be defined in the configuration.json file of Add-ins, which will then add them to the list of permissions available when editing clearances. These definitions can support multiple languages.</p>
+		<p>Administrator clearance will remain non-editable and have all custom clearances enabled by default. All other default/system clearance levels only allow editing of custom security identifiers, while keeping the system defined default identifiers non-editable.</p>
+		<p><b>* NOTE</b>: By default, once the <code className="small-code-name">enableViewSecurityId</code> property in the configuration.json for an Add-in is set to True, a View “Add-in name” add-in identifier is created that must be enabled for users to be able to view the Add-in. More granular control needs to be set within the Add-in code for any custom identifiers added to the configuration.json.</p>
+		<h2>IP Rate LimitsPermalink</h2>
+		<ul>
+			<li>
+				Added new rate limits for API requests for endpoints that do not require authentication. See <a href="https://docs.google.com/document/d/1sUKaOKjVT23qvaCITMseIMqGUhQzlr7xDhq1EjrseYE/edit#heading=h.ygvbrm3xiv4u">this announcement</a> for more information.
+			</li>
+		</ul>
+		<h2>Updates</h2>
+		<h3>DatabaseExists</h3>
+		<ul>
+			<li>
+				Removed the DatabaseExists API that was originally used for database registration purposes. If the database name exists, <code className="small-code-sample">CreateDatabase</code> throws a <code className="small-code-sample">RegistrationException</code>.
+			</li>
+		</ul>
+		<h3>RouteSearch</h3>
+		<ul>
+			<li>
+				Removed the <code className="small-code-sample">RouteSearchGroups</code> property and replaced it with the <code className="small-code-sample">ServiceGroups</code> property.
+			</li>
+		</ul>
+		<h3>PropertySelector</h3>
+		<ul>
+			<li>
+				Added the optional  <code className="small-code-sample">PropertySelector</code> parameter that is used with the <code className="small-code-sample">Get</code> and <code className="small-code-sample">GetFeed</code> methods to include or exclude specific properties for entity types requested. Detailed information can be found on the <a href="https://geotab.github.io/sdk/software/guides/concepts/#propertyselector-beta">Geotab Developers</a> site.
+			</li>
+		</ul>
+		<h3>DriverWhiteListContent / DriverAuthListContent</h3>
+		<ul>
+			<li>
+				Removed the <code className="small-code-sample">DriverWhiteListContent</code> object and replaced it with the <code className="small-code-sample">DriverAuthListContent</code> object.
```

Comment:
Content inside of code tags should be surrounded by curly brackets & ticks { `` }